crates/ide-assists/src/handlers/convert_comment_block.rs

```diff
@@ -159,15 +163,16 @@ pub(crate) fn relevant_line_comments(comment: &ast::Comment) -> Vec<Comment> {
 //              */
 //
 // But since such comments aren't idiomatic we're okay with this.
-pub(crate) fn line_comment_text(indentation: IndentLevel, comm: ast::Comment) -> String {
-    let contents_without_prefix = comm.text().strip_prefix(comm.prefix()).unwrap();
+pub(crate) fn line_comment_text(indentation: IndentLevel, comm: ast::Comment) -> Option<String> {
```

Comment:
```suggestion
pub(crate) fn line_comment_text(indentation: IndentLevel, comm: ast::Comment) -> String {
```
This is infallible now, so we can revert most of the changes here and it becomes lazy again